exporter/trace/trace_proto.go

```diff
@@ -71,22 +71,21 @@ func generateDisplayName(s *export.SpanData, format DisplayNameFormatter) string
 	}
 }
 
-
-// If there are duplicate keys present in the list of attributes, 
-// then the first value found for the key is preserved.
+// If there are duplicate keys present in the list of attributes,
+// then the last value found for the key is preserved.
 func injectLabelsFromResources(sd *export.SpanData) {
```

Comment:
Even though the logic is to preserve the first value, the result would be the last value because Resource package would overwrite the values. https://github.com/open-telemetry/opentelemetry-go/blob/v0.7.0/sdk/resource/resource.go#L40

Reply:
You're not invoking the `New` function here, so I don't understand why did you rewrite the comment. In this function the first value is preserved, no?